dipper/sources/Ensembl.py

```diff
@@ -129,69 +128,68 @@ def parse(self, limit=None):
 
         LOG.info("Done parsing files.")
 
-        return
-
-    def fetch_protein_list(self, taxon_id):
-        """
-        Fetch a list of proteins for a species in biomart
-        :param taxid:
-        :return: list
-        """
-        protein_list = list()
-        # col = self.columns['ensembl_biomart']
-        col = ['ensembl_peptide_id', ]
-
-        params = urllib.parse.urlencode(
-            {'query': self._build_biomart_gene_query(taxon_id, col)})
-        conn = http.client.HTTPConnection(ENS_URL)
-        conn.request("GET", '/biomart/martservice?' + params)
-        response = conn.getresponse()
-
-        for line in response:
-            line = line.decode('utf-8').rstrip()
-            row = line.split('\t')
-            if len(row) != len(col):
-                LOG.warning("Data error for p-list query on %d", taxon_id)
-                continue
-            protein_list.append(row[col.index('ensembl_peptide_id')])
-        conn.close()
-        return protein_list
+    # this seems so un specific as to be a bit pointless
+    # and will be deleted if there are no objections
+    # def fetch_protein_list(self, taxon_id):
+    #    """
+    #    Fetch a list of proteins for a species in biomart
+    #    :param taxid:
+    #    :return: list
+    #    """
+    #    protein_list = list()
+    #    att = ['ensembl_peptide_id', ]
+    #
+    #    params = urllib.parse.urlencode(
+    #        {'query': self._build_biomart_gene_query(taxon_id, att)})
+    #    conn = http.client.HTTPConnection(ENS_URL)
+    #    conn.request("GET", '/biomart/martservice?' + params)
+    #    response = conn.getresponse()
+    #    header = next(response).rstrip().split('\t')
+    #    for line in response:
+    #        line = line.decode('utf-8').rstrip()
+    #        row = line.split('\t')
+    #        if len(row) != len(att):
+    #            LOG.warning("Data error for p-list query on %d", taxon_id)
+    #            continue
+    #        protein_list.append(row[att.index('ensembl_peptide_id')])
+    #    conn.close()
+    #    return protein_list
```

Comment:
at one point using this in string, but we've probably moved past it, +1 for deleting unused code